cpp/ql/src/Likely Bugs/Protocols/UseOfDeprecatedHardcodedProtocol.qhelp

```diff
@@ -4,9 +4,20 @@
 <qhelp>
 <overview>
   <p>Using boost::asio library but specifying a deprecated hardcoded protocol.</p>
```

Comment:
The start of this query help is now:

"Using a deprecated hard-coded protocol using the boost::asio library.
Using boost::asio library but specifying a deprecated hardcoded protocol."

See: https://jenkins.internal.semmle.com/job/Pull-Request-Check/job/QHelp-Preview/2505/artifact/out/ql/cpp/ql/src/Likely%20Bugs/Protocols/UseOfDeprecatedHardcodedProtocol.html

I'd suggest cutting the 2nd sentence (i.e. the first p in the overview) to avoid repetition.

Reply:
Done.